sebs/openwhisk/openwhisk.py

```diff
@@ -281,27 +284,53 @@ def create_function(self, code_package: Benchmark, func_name: str) -> "OpenWhisk
                         code_package.language_name,
                         code_package.language_version,
                     )
-                    subprocess.run(
-                        [
-                            *self.get_wsk_cmd(),
-                            "action",
-                            "create",
-                            func_name,
-                            "--web",
-                            "true",
-                            "--docker",
-                            docker_image,
-                            "--memory",
-                            str(code_package.benchmark_config.memory),
-                            "--timeout",
-                            str(code_package.benchmark_config.timeout * 1000),
-                            *self.storage_arguments(),
-                            code_package.code_location,
-                        ],
-                        stderr=subprocess.PIPE,
-                        stdout=subprocess.PIPE,
-                        check=True,
-                    )
+                    if code_package.language_name == 'java':
```

Comment:
I'm wondering why we need to specify this argument for Java, and not for Python and Nodejs?